ipinfo/flags.json

```diff
@@ -0,0 +1,252 @@
+	{
+	"AD": {"emoji": "🇦🇩","unicode": "U+1F1E6 U+1F1E9"},
```

Comment:
Instead of the tab character can we use normal spaces for all of these? Even 2 is enough.